rmgpy/rmg/model.py

```diff
@@ -1404,6 +1405,7 @@ def addReactionToUnimolecularNetworks(self, newReaction, newSpecies, network=Non
 
         # Add the path reaction to that network
         network.addPathReaction(newReaction)
+        network.invalidate()
```

Comment:
This one might be unnecessary since `addPathReaction` will invalidate the network.